src/pages/validator/external-chains/centrifuge.mdx

```diff
@@ -12,22 +12,42 @@ Set up your Centrifuge Mainnet or Testnet node.
 - [Releases](https://github.com/centrifuge/centrifuge-chain/releases)
 - Docker OR `rustup` is installed
 
-## Steps
+### Optional: Using a snapshot from the Centrifuge dev team
+-> This will make your node pretty much ready to receive requests in within hours rather than days.
+
+  Using Centrifuge provided snapshots (faster)
```

Comment:
Remove this line (no longer needed because of the sentence "This will make your node pretty much ready to receive requests in within hours rather than days")

Reply:
rewrote this section to be more concise and clear